### PR TITLE
installing and configuring the registration validation module

### PR DESCRIPTION
**Pull request by katlazarova**

### CODE REVIEW

File: config/sync/registration_validation.settings.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+domains: ''
+usernames: "admin\r\nroot\r\nwebmaster\r\nwebadmin\r\npostmaster\r\nadministration\r\nadmin"
```
Comment (ElXyfer) [inline config/sync/registration_validation.settings.yml]: Is there a reason admin is added in twice and also should administrator not be added in this list?

Comment (katlazarova) [reply config/sync/registration_validation.settings.yml]: You mean it shouldn't be added in this list?

Comment (ElXyfer) [reply config/sync/registration_validation.settings.yml]: No i mean it's added twice in the list. The first and last value both say "admin".

Comment (katlazarova) [reply config/sync/registration_validation.settings.yml]: omg Lincoln I mean administration

Comment (ElXyfer) [reply config/sync/registration_validation.settings.yml]: It should be added in the list.

Comment (katlazarova) [reply config/sync/registration_validation.settings.yml]: well it is. So I'm pushing now.

Comment (katlazarova) [reply config/sync/registration_validation.settings.yml]: sorry I misread your earlier comment. It is done.

File: config/sync/registration_validation.settings.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+domains: ''
+usernames: "admin\r\nroot\r\nwebmaster\r\nwebadmin\r\npostmaster\r\nadministration\r\nadministrator"
+domains_message: 'The e-mail domain %domain has been blacklisted from registering. Please enter a different e-mail address.'
+usernames_message: 'The username contains %username, which has been blacklisted from being used when registering. Please enter a different username.'
+validation_counter: 2
+_core:
+  default_config_hash: bXfy4ebH_6Zsl-N1wtivC9-L2Js37ddprpeeDVxBVdI
```
Comment (ElXyfer) [inline config/sync/registration_validation.settings.yml]: Did you test this?

Comment (katlazarova) [reply config/sync/registration_validation.settings.yml]: I did test it and it works, except the error message doesn't appear. I'm looking into it but I'm having no luck.

Comment (ElXyfer) [reply config/sync/registration_validation.settings.yml]: So that means this pull request is not ready for tech review then?

Comment (katlazarova) [reply config/sync/registration_validation.settings.yml]: It is ready for tech review now.

File: config/sync/registration_validation.settings.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+domains: ''
+usernames: "admin\r\nroot\r\nwebmaster\r\nwebadmin\r\npostmaster\r\nadministration\r\nadministrator"
```
Comment (ElXyfer) [inline config/sync/registration_validation.settings.yml]: The user name "siteadmin" would be accepted, please use a wildcard on the word admin and any others you deem necessary

Comment (katlazarova) [reply config/sync/registration_validation.settings.yml]: done

File: config/sync/registration_validation.settings.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+domains: ''
+usernames: "*admin*\r\nroot\r\n*master*"
```
Comment (ElXyfer) [inline config/sync/registration_validation.settings.yml]: Why did you remove the other user names that used to be here?

Comment (katlazarova) [reply config/sync/registration_validation.settings.yml]: because they contain the wildcard words and therefore were redundant.

Comment (ElXyfer) [reply config/sync/registration_validation.settings.yml]: "rootuser" is a selectable username, please also turn root into a wildcard.

Comment (katlazarova) [reply config/sync/registration_validation.settings.yml]: done
